outbound.js

```diff
@@ -421,18 +423,78 @@ exports.stats = function () {
     return results;
 };
 
-function _next_uniq () {
```

Comment:
This function ensures that we don't overwrite the same file if the same thing happens on the same PID within one millisecond. You now have nothing to prevent that.